safe_control_gym/envs/gym_control/cartpole.py

```diff
@@ -422,8 +409,18 @@ def _setup_symbolic(self):
         # Define dynamics and cost dictionaries.
         dynamics = {'dyn_eqn': X_dot, 'obs_eqn': Y, 'vars': {'X': X, 'U': U}}
         cost = {'cost_func': cost_func, 'vars': {'X': X, 'U': U, 'Xr': Xr, 'Ur': Ur, 'Q': Q, 'R': R}}
+        # Additional params to cache
+        params = {
```

Comment:
why do we need this?

Reply:
ah its passed to SymbolicModel, mb

Reply:
adam mentions he wanted to move `X_EQ, U_EQ` inside the prior, and since PID need access to the prior quad mass, i just group them inside the prior

Reply:
Makes sense, ty